tsconfig.prod.json

```diff
@@ -5,7 +5,6 @@
         "moduleResolution": "node",
         "target": "es5",
         "noImplicitAny": false,
-        "allowUmdGlobalAccess": true,
```

Comment:
This is a new-ish TypeScript config value. It may come back in, but only after the TS version gets bumped in the dependencies.